ssl/quic/quic_stream_map.c

```diff
@@ -757,7 +757,7 @@ void ossl_quic_stream_map_remove_from_accept_queue(QUIC_STREAM_MAP *qsm,
     --qsm->num_accept;
 
     if ((max_streams_rxfc = qsm_get_max_streams_rxfc(qsm, s)) != NULL)
-        ossl_quic_rxfc_on_retire(max_streams_rxfc, 1, rtt);
+        (void) ossl_quic_rxfc_on_retire(max_streams_rxfc, 1, rtt);
```

Comment:
Nit: There should be no space after `(void)`.

Also could you please retarget the pull request against the master branch? This is just a cleanup commit so not eligible for stable branches.